client/test/test.py

```diff
@@ -0,0 +1,98 @@
+#import the module here
```

Comment:
Testing convention not followed.
https://fastapi.tiangolo.com/tutorial/testing/

Reply:
arent we testing the functions which are gonna hit fastapi 
But ig this conventions can be followed for server side

Reply:
Why? Conventions are there for a reason. It's not a guessing game please follow.